src/ert/storage/local_ensemble.py

```diff
@@ -576,6 +576,33 @@ def load_parameters(
 
         return self._load_dataset(group, realizations)
 
+    def load_cross_correlations(self) -> xr.Dataset:
+        input_path = self.mount_point / "corr_XY.nc"
+
+        if not input_path.exists():
+            raise FileNotFoundError(
+                f"No cross-correlation data available at '{input_path}'. Make sure to run the update with "
+                "Adaptive Localization enabled."
+            )
+        return xr.open_dataset(input_path, engine="scipy")
+
+    def save_cross_correlations(
```

Comment:
This should `@require_write`

Reply:
Do we also want this to give a `UserWarning` about experimental and log usage?

Reply:
I've added a log-statement, but not sure about the experimental warning. Are we saying this is experimental because the whole of adaptive localization is still experimental or something else?

Reply:
I just overheard a discussion about that I think?